lib/wallet/walletdb.js

```diff
@@ -194,6 +194,13 @@ class WalletDB extends EventEmitter {
       this.state.height,
       this.state.startHeight);
 
+    for (const options of this.options.createWalletOptions) {
+      const wallet = await this.ensure(options);
+      for (const accountName of options.accounts) {
```

Comment:
I'm not actually super happy with this magic `accounts` key, so this might change. 

Plus, I think we can just derive the non-`default` accounts (in derivation order) _after_ all the wallets are initialized. This was really just a quick way to check that it was working.